py/qstr.h

```diff
@@ -97,7 +112,9 @@ void qstr_pool_info(size_t *n_pool, size_t *n_qstr, size_t *n_str_data_bytes, si
 void qstr_dump_data(void);
 
 #if MICROPY_ROM_TEXT_COMPRESSION
-void mp_decompress_rom_string(byte *dst, mp_rom_error_text_t src);
+// CIRCUITPY-CHANGE: TODO not const previusly, should it be??
+// TEST COMPILE with const ******************
```

Comment:
Want to do this?

Reply:
I did :), and forgot to remove the comment.